ptclient/cts/spi/goal/cl/GoalHistoryItem.vue

```diff
@@ -23,8 +23,14 @@
 
 <script>
 import { GOAL_API_URL } from '@/static/others.js'
+import ormGoal from '@/models/Goal'
 export default {
-  props: ['goal'],
+  props: {
+    goal: {
+      type: ormGoal,
+      required: true,
+    },
+  },
```

Comment:
Since this component depends on a VuexORM Model passed down as a prop, again being detailed with component props, we can import the model and use it as the prop type for strict checking. It's set as required since the component won't function without a valid value.